cmd/corectl/main.go

```diff
@@ -68,6 +68,8 @@ var commands = map[string]*command{
 	"get":                  {get},
 	"add":                  {add},
 	"rm":                   {rm},
+	"set":                  {set},
+	"unset":                {unset},
```

Comment:
Would it be possible (and would it be a good idea) to use `rm` for this as well? Maybe even all the way in to the Remove handler?